src/pages/Transfer/CrossChainTransferForm/index.tsx

```diff
@@ -247,26 +253,25 @@ const CrossChainTransferForm = (): JSX.Element => {
     return <PrimaryColorEllipsisLoader />;
   }
 
+  const isRelayChain = fromChain === ChainType.RelayChain;
+  const balance = displayMonetaryAmount(isRelayChain ? relayChainBalance : collateralTokenTransferableBalance);
+  const availableBalanceLabel = isRelayChain
+    ? t('transfer_page.cross_chain_transfer_form.relay_chain_balance')
+    : t('transfer_page.cross_chain_transfer_form.parachain_balance');
+
   return (
     <>
       <form className='space-y-8' onSubmit={handleSubmit(onSubmit)}>
         <FormTitle>{t('transfer_page.cross_chain_transfer_form.title')}</FormTitle>
         <div>
-          {fromChain === ChainType.RelayChain ? (
```

Comment:
Not sure about this change - not necessarily a bad thing to consolidate down to a single instance of `<AvailableBalanceUi>` but I find it easier to follow as it was originally, even though it's a bit more verbose.

Reply:
On reflection, a bit more context here might be helpful :) I prefer to avoid multiple instances of the same conditional where possible. So here, this was written like this so that there could be a single check against `fromChain === ChainType.RelayChain ?` with the values handled by rendering instances of `AvailableBalanceUi`.

Reply:
The check `fromChain === ChainType.RelayChain` was occurring times in this file, so I have decided to extract it to a variable.

I need to determine which balance to use for when the user clicks and to achieve that I need to extract it to a variable. In the end, I was left we no reason to keep the old ternary render.

Reply:
Ah ok - in that case fine :) if it's being outside of the render as well then the change makes sense.